samples/oauth_rails/VetsApiAppeals/app/views/nods/new.html.erb

```diff
@@ -14,6 +14,12 @@
   let headerSchema = <%= raw @header_schema %>;
   let schema = <%=raw @schema %>;
 
+  for (const d in headerSchema['definitions']) {
+    if ( /^X-/.test(d) ) {
+      headerSchema['definitions'][d]['type'] = 'string'
```

Comment:
Not a blocker, but I feel like this is a miss on the schema's part for not defining the types. I wonder if it's even a valid schema without them...something to investigate.

Reply:
Hmm, yeah. That is a good point.